bin/NASA-Acronyms.js

```diff
@@ -0,0 +1,29 @@
+const path = require("path");
+const fetch = require("node-fetch");
+
+function checkStatus(res) {
+  if (res.ok) {
+    // res.status >= 200 && res.status < 300
```

Comment:
Should you just remove this comment? Git will retain history. This way we have cleaner code.

Reply:
I did it and opened a pull request with his changes:
[Delete remaining comments](https://github.com/nasa/NASA-Acronyms/pull/10)

It's ok? 😮